app/renderer/js/utils/config-util.js

```diff
@@ -65,26 +66,11 @@ class ConfigUtil {
 		return (value !== undefined);
 	}
 
-	setConfigItem(key, value) {
-		const item = {};
-		if (this.isConfigItemExists(key) && this.getConfigItem(key).admin) {
-			// if admin is set to true, user is not allowed to change that setting
-			return;
+	setConfigItem(key, value, override = false) {
```

Comment:
What the point of having the optional override parameter, for future use?

Reply:
While initializing default settings for the first time, the app may read a few admin-only fields from the custom config file that will need to reflect in the user's `settings.json`. To keep code simple in `setConfigItem()`, I just set the optional override parameter to true when initializing the app. Requires less/no changes elsewhere in the app that this function is used.

Reply:
Here is the code in `initDefaultSettings()`:
```javascript
if (EnterpriseUtil.isConfigItemExists(i)) {
	ConfigUtil.setConfigItem(i, EnterpriseUtil.getConfigItem(i), true);
} else {
	ConfigUtil.setConfigItem(i, settingOptions[i]);
}
```

Reply:
OK, make sense.

Reply:
Thanks for the review. I'll clean up git history and make the changes required.